db/column_family.cc

```diff
@@ -359,6 +359,10 @@ ColumnFamilyOptions SanitizeOptions(const ImmutableDBOptions& db_options,
     }
   }
 
+  if (result.blob_path.empty() && !result.cf_paths.empty()) {
+    result.blob_path = result.cf_paths.front().path;
+  }
+
```

Comment:
I worry about this approach and relocating/moving directories.  I think the approach taken in #8582 for dealing with the wal_dir would be a better solution.  Leave the value blank if not set or unset it if if matches the cf_path.  Do not force it to be set.